tests-legacy/Integration/PrestaShopBundle/Test/LightWebTestCase.php

```diff
@@ -100,27 +100,20 @@ protected function setUp()
 
         $contextMock->shop = $shopMock;
 
-        $themeMock = $this->getMockBuilder(Theme::class)
-            ->setMethods(array('getName'))
-            ->disableOriginalConstructor()
-            ->getMock();
-
-        $themeMock->method('getName')
-            ->willReturn('classic');
-
-        $contextMock->shop->theme = $themeMock;
+        $contextMock->shop->theme = new Theme([
+            'name' => 'classic',
+            'directory' => _PS_ROOT_DIR_.'/themes/',
+        ]);
 
         $countryMock = $this->getMockBuilder(Country::class)
             ->getMock();
         $countryMock->iso_code = 'en';
 
         $contextMock->country = $countryMock;
 
-        $languageMock = $this->getMockBuilder(Language::class)
-            ->disableAutoload()
-            ->disableOriginalConstructor()
-            ->getMock();
-        $contextMock->language = $languageMock;
+        $languageFixture = new Language(1);
```

Comment:
@matks is it ok to use language fixture instead of mock? This solves some problems with tests.

Reply:
Yep, it's alright. Those tests aim to check basic page loading capabilities, not prestashop core.